src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthenticationBackend.java

```diff
@@ -54,6 +54,7 @@ public class LDAPAuthenticationBackend implements AuthenticationBackend {
 
     private final Settings settings;
     private final Path configPath;
+    private final boolean shouldFollowReferrals;
```

Comment:
Note; in this file there are many static methods that cannot use the field, maybe better to remove the field.